packages/navigator/src/browser/navigator-preferences.ts

```diff
@@ -24,12 +24,18 @@ export const FileNavigatorConfigSchema: PreferenceSchema = {
             type: 'boolean',
             description: 'Selects file under editing in the explorer.',
             default: true
-        }
+        },
+        'explorer.countBadge': {
+            type: 'boolean',
+            description: 'Controls the count badge on the Explorer icon.',
```

Comment:
I would not mention `icon` but rather the view itself:

```suggestion
            description: 'Controls whether badges are enabled for the explorer.',
```

The same applies to other similar preferences.